### PR TITLE
initial face work

### PR DESCRIPTION
**Pull request by deguzmankevin**
### What does this PR do?

This PR adds much of the first part of the new profile page (aka FACE) to phoenix-next. 

### Any background context you want to provide?

This is the first actual React/Redux/GraphQL code I've ever written - please let me know if I'm doing anything seriously wrong or not using them how they're meant to be used! 

So far it's pretty [similar](https://ibb.co/e9koP8) to the wireframe and is mostly functional

### What are the relevant tickets/cards?

Refs [#157662685](https://www.pivotaltracker.com/story/show/157662685)

### CODE REVIEW

File: routes/web.php [hunk 1]
```diff
@@ -17,17 +17,23 @@
...
 
 // Non-campaign pages
-$router->get('/us/{slug}', 'PageController@show');
-$router->get('/{slug}', function ($slug) {
-    return redirect('/us/'.$slug);
-});
+// $router->get('/us/{slug}', 'PageController@show');
+// $router->get('/{slug}', function ($slug) {
+//     return redirect('/us/'.$slug);
+// });
```
Comment (weerd) [inline routes/web.php]: Commented out?

Comment (deguzmankevin) [reply routes/web.php]: oh, WIP from figuring out problems with routing

File: resources/assets/components/pages/ProfilePage/TestCampaign.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+import Enclosure from '../../Enclosure';
+
+const TestCampaign = () => (
```
Comment (deguzmankevin) [inline resources/assets/components/pages/ProfilePage/TestCampaign.js]: just a placeholder for the campaigns portion until I start the next part!

**Comment by mendelB** [comment]
This is super exciting stuff! Let the Face begin!

**Pull request merged**